addons/editor-compact/addon.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Compact editor",
+  "description": "Decreases the size of buttons, input fields, menu bars, and other elements in the editor, making more space for components such as the code workspace, the costume editor, and costume previews, as was the case in Scratch 2.0.",
+  "tags": ["editor", "theme"],
+  "credits": [
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://scratch.mit.edu/users/lisa_wolfgang/"
+    },
+    {
+      "name": "Maximouse",
+      "link": "https://github.com/mxmou/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["projects"]
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "versionAdded": "1.30.0",
```

Comment:
```suggestion
  "versionAdded": "1.32.0",
```
1.32 has been released some time ago (March 11<sup>th</sup>, 2023) so this might not work for older SA versions.

Reply:
> this might not for older SA versions

The addon would work for older SA versions. The `versionAdded` property just decides whether the addon should be shown in the `New and updated addons` category. Just clarifying your statement.

Reply:
> The addon would work for older SA versions. The `versionAdded` property just decides whether the addon should be shown in the `New and updated addons` category.

Thanks.